src/main/java/com/google/devtools/build/lib/util/Markdown.java

```diff
@@ -20,7 +20,7 @@
 /** Helpers for markdown. */
 public class Markdown {
   private static final Parser PARSER = Parser.builder().build();
-  private static final HtmlRenderer HTML_RENDERER = HtmlRenderer.builder().build();
+  private static final HtmlRenderer HTML_RENDERER = HtmlRenderer.builder().escapeHtml(true).build();
```

Comment:
our internal tooling reported that the tests still pass if this is changed to `escapeHtml(false)`. Could you double-check whether that's accurate and whether any tests need to be updated?

Reply:
I'll double check but I don't believe they are any existing tests validating HTML escape logic. Best there is for now is diffing command line docs manually (I believe there is some syntax that renders funny with the existing setup).

Otherwise I'm happy to add test coverage for this. Though it may take some time (busy few weeks).

Reply:
ok, that's fine. I'll proceed with the merge for now so that it makes the 9.0 cut. If anything needs fixing, we can cherry-pick.

Reply:
I had a moment and have raised #27488 to address the test coverage gap.